src/encoded/ingestion/processors.py

```diff
@@ -139,6 +139,7 @@ def handle_variant_update(submission: SubmissionFolio):
 
 
 @ingestion_processor('metadata_bundle')
+@ingestion_processor('family_history')
```

Comment:
If I'm understanding right, you're really trying to rename the `'metadata_bundle'` type to `'accessioning'`.  It would make sense in that case to add a third item here.
```suggestion
@ingestion_processor('metadata_bundle')  # obsolete. use 'accessioning'.
@ingestion_processor('accessioning')
@ingestion_processor('family_history')
```
Also, I've here gone ahead and used your term (`'accessioning'`), but I'd _much_ rather see you use `'accession'` instead. It's just cleaner and I don't think using the gerund form really adds anything. "accession" is only a verb by our abuse of the language, and I kinda would like to minimize that where not needed.

Reply:
I'd really rather keep this as "accessioning" for now because it matches the terminology that appears multiple times on the case page. The concept of "abuse" of language is really rather meaningless because the nature of language is to constantly change, but I won't go into that here. I do think it's worthwhile to use terms that users and clinicians will be familiar with, and I hear "accessioning" talked about often but never an "accession" mentioned unless it is a specific identifier of an item (and this metadata is much more than just that).